ui/app/templates/components/dashboard/replication-card.hbs

```diff
@@ -0,0 +1,127 @@
+<div class="is-flex-between">
+  <h3 class="title is-4 has-bottom-margin-xxs" data-test-client-count-title>
+    Replication
+  </h3>
+
+  <div>
+    <Hds::Link::Inline @route="vault.cluster.replication.index" class="is-no-underline">Details</Hds::Link::Inline>
+  </div>
+</div>
+{{! check if dr replication and performance replication exists }}
+{{#if (or @replication.dr.clusterId @replication.performance.clusterId)}}
+  {{! check if user has access to both perf replication and dr replication }}
+  {{#if (and @version.hasPerfReplication @version.hasDRReplication)}}
+    <hr class="has-background-gray-100" />
+    <div
```

Comment:
@hashishaw I agree! Do you have suggestions on where to put the `display-only` components? Debating if it should live in a level as the other dashboard components

Reply:
I think alongside the dashboard components makes sense! depending on how generic they are you could name them something sort of generic like `text-lockup` or since there are `running` and `not running` states you might name it `replication-text`

Reply:
I just realized we can use the `StatText` component. I modified the `StatText` component to use [named-blocks](https://guides.emberjs.com/v3.27.0/components/block-content/). I think the `replication-card` template is still very long, and could probably be more modular. The only downside is that those wouldn't be generic. Open to feedback here!

Reply:
Looking at the changes made below to `StatText` it feels like the component may be deviating a lot from it's intended use case - especially since we're not rendering a statistic/numerical value here. 

I'd suggest making a new component that handles all of the styling/divs for you so all you have to render here is something like: 

```
<Replication::StateCard
 @title="DR Primary"
 @subtext=".....
 @status={{get (cluster-states 
 @replication.dr.state)}} 
 />
<Replication::StateCard
 @title="Perf Primary"
 @subtext=".....
 @status={{get (cluster-states 
 @replication.performance.state)}}
 />
 ```
 
 That way we can also avoid extra, blank `<div>` (like on line 20) which I think helps screen readers so they aren't skipping through divs on divs

Reply:
Example:
```html
<div class="has-bottom-margin-s">
  <Hds::Link::Inline
    class="title is-5 has-text-weight-semibold"
    @route="vault.cluster.replication.mode.index"
    @model={{@name}}
    data-test-title={{@name}}
  >
   {{@title}}
  </Hds::Link::Inline>
</div>
<div class="title is-4 has-text-weight-semibold has-top-margin-xs" data-test-state={{@name}}>
  {{@state}}
  <Hds::TooltipButton @text="The cluster's current operating state">
    <FlightIcon
      @name={{get (cluster-states @state) "glyph"}}
      class={{if (get (cluster-states @replication.dr.state) "isOk") "has-text-success" "has-text-danger"}}
    />
  </Hds::TooltipButton>
</div>

Reply:
@hellobontempo  I'm not sure about creating a `Replication::StateCard` component since the styles are the same and arguments are mostly the same as the `StatText` component. The difference is the title link + the extra icon for some of the states.

If we change the title to be non-links (I still need to confirm with the design if we want to do that), it's basically the `StatText` component minus the icon/tooltip that can be yielded at the end of the component. I do understand that the data passed to the `StatText` component isn't necessarily stats to be displayed on the page and the title links cause a difference in the StatText component, so I could see why we might do that! The other thing about creating a `Replication::StateCard` component is that we also would copy pasta all the styles from the `stat-text.scss` since those styles are scoped to `StatText`?